packages/i18n/src/translateToExclusiveList.js

```diff
@@ -13,6 +13,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+
```

Comment:
?

Reply:
I don't know why this line here specifically got changed, but I don't mind it.

Having some breathing room here after the license text and around docblocks makes sense to me.